contract-examples/contracts/ExampleFeeManager.sol

```diff
@@ -84,3 +103,131 @@ contract ExampleFeeManager is AllowList {
+
+    FeeConfig memory newFeeConfig = FeeConfig({
+      gasLimit: CCHAIN_GAS_LIMIT,
+      targetBlockRate: CCHAIN_TARGET_BLOCK_RATE,
+      minBaseFee: CCHAIN_MIN_BASE_FEE,
+      targetGas: CCHAIN_TARGET_GAS,
+      baseFeeChangeDenominator: CCHAIN_BASE_FEE_CHANGE_DENOMINATOR,
+      minBlockGasCost: CCHAIN_MIN_BLOCK_GAS_COST,
+      maxBlockGasCost: CCHAIN_MAX_BLOCK_GAS_COST,
+      blockGasCostStep: CCHAIN_BLOCK_GAS_COST_STEP
+    });
+
+    assertNotEq(config.gasLimit, newFeeConfig.gasLimit);
+    // target block rate is the same for wagmi and cchain
+    // assertNotEq(config.targetBlockRate, newFeeConfig.targetBlockRate);
+    assertNotEq(config.minBaseFee, newFeeConfig.minBaseFee);
+    assertNotEq(config.targetGas, newFeeConfig.targetGas);
+    assertNotEq(config.baseFeeChangeDenominator, newFeeConfig.baseFeeChangeDenominator);
+    // min block gas cost is the same for wagmi and cchain
+    // assertNotEq(config.minBlockGasCost, newFeeConfig.minBlockGasCost);
+    assertNotEq(config.maxBlockGasCost, newFeeConfig.maxBlockGasCost);
+    assertNotEq(config.blockGasCostStep, newFeeConfig.blockGasCostStep);
+
+    example.enableCChainFees();
+
+    FeeConfig memory changedFeeConfig = example.getCurrentFeeConfig();
+
+    assertEq(changedFeeConfig.gasLimit, newFeeConfig.gasLimit);
+    // target block rate is the same for wagmi and cchain
+    // assertEq(changedFeeConfig.targetBlockRate, newFeeConfig.targetBlockRate);
+    assertEq(changedFeeConfig.minBaseFee, newFeeConfig.minBaseFee);
+    assertEq(changedFeeConfig.targetGas, newFeeConfig.targetGas);
+    assertEq(changedFeeConfig.baseFeeChangeDenominator, newFeeConfig.baseFeeChangeDenominator);
+    // min block gas cost is the same for wagmi and cchain
+    // assertEq(changedFeeConfig.minBlockGasCost, newFeeConfig.minBlockGasCost);
+    assertEq(changedFeeConfig.maxBlockGasCost, newFeeConfig.maxBlockGasCost);
+    assertEq(changedFeeConfig.blockGasCostStep, newFeeConfig.blockGasCostStep);
+
+    assertEq(example.getFeeConfigLastChangedAt(), block.number);
+
+    example.enableCustomFees(config);
+  }
+
+  function test_minFeeTransaction() public {
+    // noop
+  }
+
+  function test_raiseMinFeeByOne() public {
+    ExampleFeeManager example = new ExampleFeeManager();
+    address exampleAddress = address(example);
+
+    IFeeManager manager = IFeeManager(FEE_MANAGER_ADDRESS);
+
+    manager.setEnabled(exampleAddress);
+
+    FeeConfig memory config = example.getCurrentFeeConfig();
+    config.minBaseFee = config.minBaseFee + 1;
+
+    example.enableCustomFees(config);
+  }
+
+  function test_lowerMinFeeByOne() public {
+    ExampleFeeManager example = new ExampleFeeManager();
+    address exampleAddress = address(example);
+
+    IFeeManager manager = IFeeManager(FEE_MANAGER_ADDRESS);
+
+    manager.setEnabled(exampleAddress);
+
+    FeeConfig memory config = example.getCurrentFeeConfig();
+    config.minBaseFee = config.minBaseFee - 1;
+
+    example.enableCustomFees(config);
```

Comment:
see here
https://github.com/ava-labs/subnet-evm/pull/601/files#r1178095800

Reply:
shouldn't we assert that (and on above) that `getCurrentFeeConfig().minBaseFee ==  config.minBaseFee - 1`?

Also IMO `test_` prefix sounds a bit weird when we don't actually test anything :/

Reply:
See [my comment here](https://github.com/ava-labs/subnet-evm/pull/601#discussion_r1180778950). `test_` implies a test context, not that we are necessarily testing something. 

If there's a prefix that would be more suitable, I'm open to suggestions. 

And I don't think we need to check to see if the fee is lowered. We already have tests that check to ensure we can change the fee properly.  If that's not working for whatever reason, the other test should be failing.

Reply:
Should we refer to these functions as `testStep_` or `testTx_` or `step_` instead of `test_`? 

`test_` implies that this function is a full test in most testing frameworks.

Reply:
https://github.com/ava-labs/subnet-evm/pull/636